DirectProgramming/DPC++FPGA/Tutorials/Features/fp_pragmas/sample.json

```diff
@@ -0,0 +1,55 @@
+{
+  "guid": "7909FAE1-D3D4-4E97-A963-14A884F33495",
+  "name": "FP Pragmas",
```

Comment:
Please change to Floating Point Pragmas.  

In fact, I don't really like the directory name 'fp_pragmas', what do you think of changing it to 'floating_point_pragmas'?

Reply:
Do we need to change all the occurrence of 'fp_pragmas' to 'floating_point_pragmas'?